tests/Regression.hs

```diff
@@ -3,14 +3,16 @@
 
 module Main (main) where
 
+import qualified Numeric.AD.Mode.Forward as F
+import qualified Numeric.AD.Mode.Forward.Double as FD
 import qualified Numeric.AD.Mode.Reverse as R
 import qualified Numeric.AD.Mode.Reverse.Double as RD
 
 import Text.Printf
 import Test.Tasty
 import Test.Tasty.HUnit
 
-type Diff = (forall a. Floating a => a -> a) -> Double -> Double
+type Diff = (forall a. Floating a => a -> a) -> Double -> (Double, Double)
```

Comment:
When I first read this, I was confused about why `Diff` had its return type changed to `(Double, Double)`. That's because all of its use sites have been changed from `diff` to `diff'`. To match this, perhaps it would be worth changing this type synonym's name from `Diff` to `Diff'`.